microgen/shape/strut_lattice/body_centered_cubic.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import numpy.typing as npt
+import math as m
+from itertools import product
+from .abstract_lattice import AbstractLattice
+
+
+class BodyCenteredCubic(AbstractLattice):
+    """
+    Class to create a unit body-centered cubic lattice of given cell size and density or strut radius
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs, strut_heights=m.sqrt(3.0) / 2.0)
```

Comment:
`mypy` is not happy with the current code: `"__init__" of "AbstractLattice" gets multiple values for keyword argument "strut_heights"`. I propose this change but I'm not sure how to deal properly with this.
```suggestion
	if "strut_heights" not in kwargs:
            kwargs["strut_heights"] = m.sqrt(3.0) / 2.0
        super().__init__(*args, **kwargs)
```

Reply:
Same suggestion in the other files where mypy raises this issue.

Reply:
I have added a tentative fix, but I am also not sure it is the best way of dealing with this
Popped the relevant kwargs, set default kwargs to the desired `strut_heights`, or `base_vertices` and `strut_vertex_pairs` in the case of `CustomLattice`